### DISCUSSION TITLE
Drag bounds?

### DISCUSSION THREAD

**Discussion by sami616** [pmndrs/use-gesture#56] [Q&A] [answered]
Hi i'm trying to implement some bounds for an x value when dragging an element.

Basically i need the bounds to be 100px from the sides of the window.

I have a codesandbox where i'm partly there but cannot resume dragging after the event has been cancelled. Is this the recommended way to implement this type of behaviour? I couldnt see anything in the docs about this.

https://codesandbox.io/s/ko68lqp9pv

Thanks in advance!

**Comment by dbismut** [comment, marked as answer]
Hey I've just updated the readme with an example that handles boundaries.
https://codesandbox.io/s/r7xnzk4x0o

Again, using temp is the way to go.

**Comment by sami616** [comment]
Amazing, thank you so much for your help @dbismut
